doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Materialize Open CDC"
+description: "The Open CDC format allows Materialize to detect when records are duplicated or out of order."
+menu:
+  main:
+    parent: "overview"
+    weight: 4
+---
+
+Change data capture (CDC) tools provide feeds that record any changes to a database. Typically, the feeds are then saved to another platform, like Kafka, for storage or processing.
+
+However, sometimes the stream can have missing or duplicate records, or send records out of order. For example, if a CDC tool crashes while writing a record, it may retry and write the record again, resulting in a duplicate entry.
```

Comment:
Our format isn't robust to missing records -- only to duplicate or out-of-order.